CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 2.2.0-beta4
+- Support of performance collection and live metrics for Azure Web Apps is enabled.
+- Custom performance metric support is not supported for Azure Web Apps.
```

Comment:
You do not need statements like this in release notes. Please rephrase the first statement to "default counters" collection. No need to mention custom.

BTW, why custom are not supported?